css/playback_controls.css

```
@@ -0,0 +1,108 @@
+.playback-controls {
+  align-items: center;
+  container-type: inline-size;
+  display: flex;
+  gap: 3px;
+  width: 100%;
+}
+
+.playback-controls .btn {
+  align-items: center;
+  background-color: var(--dark-one);
+  border: none;
+  color: var(--white-one);
+  display: flex;
+  justify-content: center;
+  height: 32px;
+  min-width: 32px;
+}
+
+.playback-controls .disabled svg path {
+  fill: var(--gray-four);
+}
+
+.playback-controls .btn:hover {
+  background-color: var(--gray-one);
+  border: none;
+  color: var(--white-one);
+}
+
+#back-button:hover,
+#play-button:hover,
+#next-button:hover {
+  background-color: var(--gray-one);
+  border: none;
+  color: var(--white-one);
+}
+
+#play-button {
+  padding: 0px;
+}
+
+.playback-controls .slider {
+  flex: 3;
+  margin: auto 12px;
+}
+
+.playback-controls .slider .ant-slider-track {
+  background-color: var(--white-one);
+}
+
+.playback-controls .slider .ant-slider-track,
+.playback-controls .slider .ant-slider-rail {
+  border-radius: 3px;
+}
+
+.playback-controls .slider .ant-slider-rail {
+  background-color: var(--dark-four);
+}
+
+.playback-controls .slider .ant-slider-rail:hover {
+  background-color: var(--white-one);
+}
+
+.ant-slider-handle,
+.ant-slider-handle::before,
+.ant-slider-handle::after {
+  background-color: var(--white-one);
+  border-radius: 50%;
+  box-shadow: none !important;
+  inset-block-start: 0px;
+}
+
+.time-input-display {
+  background-color: var(--dark-one);
+  border-radius: 3px;
+  height: 26px;
+  padding: 4px 5px 4px 5px;
```

```suggestion
  padding: 4px 5px;
```
Nit: Can just specify two dimensions to do the same thing!